usrsctplib/netinet/sctp_callout.c

```diff
@@ -49,10 +49,12 @@
 #include <user_atomic.h>
 #include <netinet/sctp_sysctl.h>
 #include <netinet/sctp_pcb.h>
+#include <netinet/sctp_var.h>
 #else
 #include <netinet/sctp_os.h>
 #include <netinet/sctp_callout.h>
 #include <netinet/sctp_pcb.h>
+#include <netinet/sctp_var.h>
```

Comment:
I think this is not needed. Can you remove it?

Reply:
Done, and rebased.